### PR TITLE
Update date formatting for MILESTONE_MESSAGE

### PR DESCRIPTION
**Pull request by AllegroFox**

### CODE REVIEW

File: cfr.yaml [hunk 1]
```diff
@@ -32,6 +32,34 @@
     id: contentBlocking
     params:
       - ContentBlockingMilestone
+- id: MILESTONE_MESSAGE_2
```
Comment (piatra) [inline cfr.yaml:35]: This should instead reference the Firefox version it targets, that's what our current convention looks like.

**Review by piatra — CHANGES_REQUESTED** [review]
This looks good, note that we need to update the targeting on the previous message to stop showing in 87 and later

File: cfr.yaml [hunk 2]
```diff
@@ -32,6 +32,34 @@
...
+    category: cfrFeatures
+    heading_text:
+      string_id: cfr-doorhanger-milestone-heading2
+    layout: short_message
+    notification_text: ''
+    skip_address_bar_notifier: true
+    text: ''
+  frequency:
+    lifetime: 7
+  targeting: pageLoad >= 4 && firefoxVersion >= 87
```
Comment (piatra) [inline cfr.yaml:57]: Can you also please add `userPrefs.cfrFeatures` to this and the old message. Lets not show it if users opted out.

File: cfr.yaml (hunk 1 above)
Comment (AllegroFox) [reply cfr.yaml:35]: Oh yep, I see

File: cfr.yaml [hunk 3]
```diff
@@ -26,12 +26,40 @@
...
   trigger:
     id: contentBlocking
     params:
       - ContentBlockingMilestone
+- id: MILESTONE_MESSAGE_87
+  groups:
+    - cfr
+  content:
+    anchor_id: tracking-protection-icon-box
+    bucket_id: CFR_MILESTONE_MESSAGE
```
Comment (piatra) [inline cfr.yaml:40]: bucket_id should match message id

**Review by piatra — APPROVED** [review]

File: cfr.yaml (hunk 3 above)
Comment (AllegroFox) [reply cfr.yaml:40]: oh shoot. so close!

**Pull request merged**